.github/workflows/ci.yml

```diff
@@ -33,10 +42,18 @@ jobs:
   smokes-labeler:
     name: Smoke Test Label
     runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      issues: write
```

Comment:
These permissions match the top-level, so these are just redundant. Let's remove them here.